model-optimizer/extensions/front/Mish_fusion.py

```diff
@@ -0,0 +1,59 @@
+"""
+ Copyright (C) 2018-2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+from extensions.front.Softplus_fusion import SoftplusFusion
+from extensions.ops.activation_ops import Mish
+from mo.front.common.replacement import FrontReplacementSubgraph
+from mo.front.subgraph_matcher import SubgraphMatch
+from mo.graph.graph import Graph, rename_nodes
+
+
+class MishFusion(FrontReplacementSubgraph):
+    """
+    The transformation looks for the pattern with Softplus defining the Mish function: Mish(x) = x * tanh(Softplus(x)).
+    """
+    enabled = True
+
+    def run_after(self):
+        return [SoftplusFusion]
+
+    def pattern(self):
+        return dict(
+            nodes=[
+                ('input', dict()),
```

Comment:
why do we need it?

Reply:
Valid point. I will try to avoid this.
It is used such a way because we need to make sure that the tensor is produced from the same node and goes to both `Mul` and `Softplus`